test/e2e_node/cpu_manager_metrics_test.go

```diff
@@ -189,7 +189,7 @@ func makeGuaranteedCPUExclusiveSleeperPod(name string, cpus int) *v1.Pod {
 							v1.ResourceMemory: resource.MustParse("64Mi"),
 						},
 					},
-					Command: []string{"sh", "-c", "sleep", "1d"},
+					Command: []string{"/bin/sleep", "86400"},
```

Comment:
I prefer the 1d versus 86400. Any reason for the change?

Reply:
yes, apparently the image changed and is not accepting the old value. I'll handle this on a separate PR, though.